src/utils/toasts.ts

```diff
@@ -9,22 +10,40 @@ const DEFAULT_TOAST_PROPS = {
   isClosable: true,
 }
 
-export const useErrorToast = (): UseToastReturn =>
-  useToast({
+// Not specifying return type because type is in the argument
+const useIsomerToast = (args: UseToastProps) => {
```

Comment:
wah I think we need to define a return type here, but fell into a rabbit hole for this cuz we got a version mismatch between chakra and design system. 
specifically, `danger` doesnt exist anymore, should be `error` now, but design system is lagging behind, actually not sure if this is already a bug in prod :sadge: 

Rather than putting this, could you create a ticket and assign it to me? is related to the design system changes